topics/sequence-analysis/tutorials/mapping/slides.html

```diff
@@ -964,7 +964,7 @@
 
 *“... there is no tool that outperforms all of the others in
 all the tests. Therefore, the end user should clearly
-specify his needs in order to choose the tool that
+specify [her/]his needs in order to choose the tool that
```

Comment:
```suggestion
specify [their] needs in order to choose the tool that
```

Reply:
@nsoranzo if we use `their` then we should also modify at the start of the sentence which reads as `the end user ...` and change to `the end users ...`

Reply:
@anuprulez No, it's a [singular their](https://en.wikipedia.org/wiki/Singular_they).

Reply:
@nsoranzo thanks for your suggestion :)